hostap-patches/Support-for-SUITEB192.patch

```diff
@@ -0,0 +1,220 @@
+diff --git a/src/crypto/tls_wolfssl.c b/src/crypto/tls_wolfssl.c
+index b4f1bbe..454f288 100644
+--- a/src/crypto/tls_wolfssl.c
++++ b/src/crypto/tls_wolfssl.c
+@@ -1292,6 +1292,25 @@ int tls_connection_set_params(void *tls_ctx, struct tls_connection *conn,
+ {
+ 	wpa_printf(MSG_DEBUG, "SSL: set params");
+ 
++#ifdef CONFIG_SUITEB192
++    {
++        int setErr = 0;
++	    setErr = wolfSSL_set_cipher_list(conn->ssl,
++            "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-=GCM-SHA384:"
++            "DHE-RSA-AES256-GCM-SHA384:TLS13-AES256-GCM-SHA384");
++        if (setErr == WOLFSSL_SUCCESS)
++            setErr = wolfSSL_SetMinEccKey_Sz(conn->ssl, 384);
++        if (setErr == WOLFSSL_SUCCESS)
++            setErr = wolfSSL_SetMinRsaKey_Sz(conn->ssl, 3072);
++        if (setErr == WOLFSSL_SUCCESS)
++            setErr = wolfSSL_SetMinDhKey_Sz(conn->ssl, 3072);
++
++        if (setErr != WOLFSSL_SUCCESS) {
++		    return -1;
++        }
++    }
++#endif
++
```

Comment:
It would be better to move this section to the `else` of the `if (os_strcmp(params->openssl_ciphers, "SUITEB192") != 0)` statement. If not set then this would be always applied.